### ISSUE TITLE
Remove the imgAlt prop from IconLink components for better accessibility

### ISSUE THREAD

**Issue by toolness** [mozilla/teach.webmaker.org#906]
As mentioned in #905, the [W3C Techniques for providing useful text alternatives](http://dev.w3.org/html5/alt-techniques/#sec5) states:

> Purely decorative images must be marked up so they can be ignored by assistive technology with a null alt attribute (`alt=""`) or preferably use CSS techniques. If the image isn't providing the user any informative content or enhancing greater understanding of the content, then the `alt` attribute must be empty.

The icons in our `IconLink` components are purely decorative and should be ignored by assistive technology, so we should remove the `imgAlt` prop from them and simply hard-code `alt=""` for the icons in the component's `render()` method.
